config.py

```diff
@@ -5,7 +5,7 @@
 logging.basicConfig(level=env_loglevel)
 logg = logging.getLogger(__name__)
 
-VERSION = '2.0.5'  # Remember to bump this in every PR
+VERSION = '2.0.6'  # Remember to bump this in every PR
```

Comment:
Shouldn't have any backend bump.

Should be a minor bump on frontend `2.1.0` in package.json